beta/src/components/header/index.ts

```diff
@@ -433,14 +433,16 @@ class Header extends Component<HeaderProps> {
   }
 
   renderSearch (): HTMLElement {
+    const searchClasses: string = 'search flex-l flex-auto-l w-100-l justify-center-l'
```

Comment:
Maybe I'm missing something, but what's the reason for adding the same classes to both the button and the div wrapping the button?

Reply:
I made these changes to expand the click hitbox for the Search after it's been closed, before it was hard to re-activate the search before after it was closed, you couldn't click very far above or below the icon/text nor far left or right. There may be a simpler way to make that change without doubling the classes up though. I'll investigate.

Reply:
Perhaps just giving the button some extra padding? Haven't experimented myself but it's a thought 🙂

Reply:
We definitely want the button to occupy all of the space the div occupies (if you test out this MR you'll see what I'm talking about), it's an unintuitively-narrow area you can click as it is currently if you close the search by pressing the x on the right side. I will experiment to see if I can consolidate things though!

Reply:
Hmm okay fair enough. I think the correct setup would perhaps be this then:
```
div: search flex-l justify-center-l w-100-l
button: js bn dn db-l bg-transparent flex-l justify-center-l w-100-l
```
With this I get the desired UX/UI.

I'm assuming the `search` class should be specific to the button (since that's where it was used before) and the `flex-auto-l` class shouldn't be needed.

Reply:
Also, to nag, maybe make sure the cursor is a pointer over this clickable area as well 😄

Reply:
Good thinking! Cursor addressed here: https://github.com/resonatecoop/stream/pull/216/commits/e109eced12a77cde438cfa72cb3f7c82a5cbc640, and the former points are addressed here: https://github.com/resonatecoop/stream/pull/216/commits/45c7e7cf4dffd8596f586d8d6ee5befa740052aa